modules/ROOT/pages/ldap/ldap-connector-reference.adoc

```diff
@@ -34,17 +35,17 @@ Release Notes: xref:release-notes::connector/ldap-connector-release-notes-mule-4
 | Password a| String |  |  |
 | Authentication a| String |  |  |
 | URL a| String |  |  | x
-| Reconnection a| Reconnection |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy |  |
+| Reconnection a| Reconnection |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
 | Type | Enumeration, one of:
 
-** JNDI |  The implementation of the connection to be used. |  JNDI |
-| Schema Enabled a| Boolean |  If set to true, the LDAP connector uses the LDAP schema (only works for LDAP v3) to define the structure of the LDAP entry (or map). This needs to be 'true' in order to use DataSense as it affects the implementing class of org.mule.module.ldap.api.LDAPEntry attributes. |  false |
+** JNDI |  The implementation of the connection to use. |  JNDI |
+| Schema Enabled a| Boolean |  If set to true, the LDAP connector uses the LDAP schema (only works for LDAP v3) to define the structure of the LDAP entry (or map). This needs to be 'true' to use DataSense as it affects the implementing class of org.mule.module.ldap.api.LDAPEntry attributes. |  false |
```

Comment:
A few suggestions:

Remove the parentheses around "or map."
Change "needs to be" to "must be."
Change "as it" to "because."